.github/workflows/label.yml

```
@@ -17,6 +17,11 @@ jobs:
       pull-requests: write
 
     steps:
+    - uses: actions/checkout@v4
+      with:
+        repository: ${{ github.event.pull_request.head.repo.full_name }}
+        ref: ${{ github.event.pull_request.head.ref }}
     - uses: actions/labeler@v4
```

actionlint (1.7.7)

the runner of "actions/labeler@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)